docs/content/SRS/SRS.md

```diff
@@ -261,23 +379,45 @@ _The following Software Requirements Specification for `tinymorph` is using [Vol
 
 ### 15.1 Access Requirements
 
-[Insert your content here.]
+SR-AC1. The application will use secure environment variables to manage configuration settings that include sensitive information.
```

Comment:
do we even have any env variables for users to setup?

even for configuration and most things shouldn't it just save locally on users device?

ig for our server we might want to do some related security feature?

Reply:
Removed the requirement and set it as N/A.